lib/doorkeeper/orm/active_record.rb

```diff
@@ -25,11 +25,11 @@ def self.check_requirements!(_config)
              Doorkeeper::Application.table_name
            )
           unless Doorkeeper::Application.new.attributes.include?("scopes")
-            fail <<-MSG.squish
+            puts <<-MSG.squish
 [doorkeeper] Missing column: `oauth_applications.scopes`.
-Run `rails generate doorkeeper:application_scopes
-&& rake db:migrate` to add it.
+Create the following migration and run `rake db:migrate`.
             MSG
+            puts File.read(File.expand_path('../../../generators/doorkeeper/templates/add_scopes_to_oauth_applications.rb', __FILE__))
```

Comment:
Line is too long. [134/80]<br>Do not write to stdout. Use Rails' logger if you want to log.